src/net/sourceforge/kolmafia/webui/RelayLoader.java

```diff
@@ -36,11 +37,27 @@ protected RelayLoader(final String location, final boolean isRelayLocation) {
     }
   }
 
+  private void waitForSVNUpdateToFinish() {
+    // This first loop is to wait for LoginManager to start processing updates, if it is going to
+    PauseObject pauser = new PauseObject();
+    for (int i = 0; i < 50 && !RelayServer.isRunning(); ++i) {
+      pauser.pause(200);
```

Comment:
There are a bunch of (repeated) magic numbers here, it might be good to pull them out to named static props on the class or something

Reply:
Moved them to named constants.  Do I want to make numTries a preference or is this too niche to bother?